### PR TITLE
No def plus template

### PR DESCRIPTION
**Pull request by bucaran**

### CODE REVIEW

File: pkg/omf/cli/omf_new.fish [hunk 1]
```diff
@@ -3,7 +3,7 @@ function omf_new -a option name
     case "p" "pkg" "pack" "packg" "package"
       set option "pkg"
     case "t" "th" "the" "thm" "theme" "themes"
-      set option "themes"
+      set option "theme"
```
Comment (bpinto) [inline pkg/omf/cli/omf_new.fish]: Isn't it on the `themes` folder?

Also noticed something else: https://github.com/fish-shell/oh-my-fish/blob/no-def-plus-template/pkg/omf/cli/omf_new.fish#L26

Comment (bucaran) [inline pkg/omf/cli/omf_new.fish]: Changed. This is the directory where the theme template is.

Comment (bucaran) [inline pkg/omf/cli/omf_new.fish]: @bpinto You were right. This is a bug I just put in. Silly me. Fix shortly.

File: bin/install [hunk 2]
```diff
@@ -103,7 +103,6 @@ omf_install() {
   if [ ! -d "${OMF_CONFIG}" ]; then
     echo "Writing Oh My Fish configuration → ${OMF_CONFIG}"
     mkdir -p "${OMF_CONFIG}"
-    test -f "${OMF_CONFIG}/theme"     || echo default > "${OMF_CONFIG}/theme"
```
Comment (bucaran) [inline bin/install]: So, double check; we want this right?

Comment (bpinto) [inline bin/install]: Yep.

Comment (bpinto) [inline bin/install]: For now.

Comment (bucaran) [inline bin/install]: :+1:

Comment (bpinto) [inline bin/install]: For consistency with the theme removal below, you could do: `echo "" > $OMF_CONFIG/theme`

Comment (bucaran) [inline bin/install]: or just `touch $OMF_CONFIG/theme`
:+1:

**Comment by bucaran** [comment]
LGTM?

**Comment by bpinto** [comment]
:+1:

**Pull request merged**